### PR TITLE
Feat : Added Individual Video Page and Likes | Dislikes

### PR DESCRIPTION
**Pull request by srejitk**
The following merge request adds the following features:

1. Individual Video Page
2. Likes | Dislikes
3. Liked Video Page
4. History Page 
5. Code Refactor

### CODE REVIEW

File: src/components/index.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
+import ActionBtn from "./ActionBtn/ActionBtn";
 import Header from "./Header/Header";
 import Sidebar from "./Sidebar/Sidebar";
 import Toast from "./Toast/Toast";
 import VideoCard from "./VideoCard/VideoCard";
 
-export { Toast, Header, VideoCard, Sidebar };
+export { Toast, Header, VideoCard, Sidebar, ActionBtn };
```
Comment (rahulcs754) [inline src/components/index.jsx:7]: no need to this line you can replace with import to export

File: src/context/Video/Watchlater.jsx [hunk 2]
```diff
@@ -10,15 +11,15 @@ export const getWatchlater = async (videoDispatch) => {
...
   } catch (error) {
     Toast({ message: "Couldn't fetch Watchlist", type: "error" });
     console.log(error);
   }
 };
 
-export const addToWatchlater = async (video, videoDispatch) => {
+export const addToWatchlater = async (video, userDispatch) => {
   try {
     const response = await axios.post(
```
Comment (rahulcs754) [inline src/context/Video/Watchlater.jsx:24]: you can do here 
const { status, data } = await axios.post(...);

so you can no need 34 line

File: src/context/Video/Watchlater.jsx [hunk 3]
```diff
@@ -32,16 +33,18 @@ export const addToWatchlater = async (video, videoDispatch) => {
...
-    Toast({ message: "Couldn't add to Watchlist", type: "error" });
+    toast.error("Something went wrong");
     console.log(error);
   }
 };
 
-export const removeFromWatchlater = async (video, videoDispatch) => {
+export const removeFromWatchlater = async (video, userDispatch) => {
   try {
     const response = await axios.delete(`/api/user/watchlater/${video._id}`, {
```
Comment (rahulcs754) [inline src/context/Video/Watchlater.jsx:49]: same above

**Review by rahulcs754 — APPROVED** [review]
the code looks good and clean
just small suggestion for future project

File: src/pages/PlayVideo/PlayVideo.jsx [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+            </div>
+            {/* <div
+              onClick={() => addToWatchlater(videoData, videoDispatch)}
+              className="flex-column-wrap flex-mid-center"
+            >
+              <button className={`btn btn_action  `}>
+                <span className={`material-icons`}>playlist_add</span>
+              </button>
+              <p className="subtitle-1">Watch Later</p>
+            </div> */}
```
Comment (nakulsharma15) [inline src/pages/PlayVideo/PlayVideo.jsx:136]: You can remove comments if it's no longer needed.

**Review by nakulsharma15 — APPROVED** [review]
Well written code. No changes suggested. I liked how you named the variables and classes.

**Comment by srejitk** [comment]
Thank You.

File: src/components/index.jsx (hunk 1 above)
Comment (srejitk) [reply src/components/index.jsx:7]: Thank You. Will do that during refactoring.

File: src/context/Video/Watchlater.jsx (hunk 3 above)
Comment (srejitk) [reply src/context/Video/Watchlater.jsx:49]: Thank You. Will do that during refactoring.

File: src/pages/PlayVideo/PlayVideo.jsx (hunk 4 above)
Comment (srejitk) [reply src/pages/PlayVideo/PlayVideo.jsx:136]: Yes, thank You. Will be removing them.

**Pull request merged**